docs/specviz2d/plugins.rst

```diff
@@ -78,11 +79,16 @@ Trace objects created outside of jdaviz can be loaded into the app via :py:meth:
 
 and then added to the viewer through the data menu.
 
-Once trace objects are loaded into the app, they can be offset (in the cross-dispersion direction)
-by selecting the trace label, entering an offset, and overwriting the existing data entry (or
-creating a new one) with the modified trace.
+Suppose you have 2D spectra of an extended source, and you have already created a trace that follows the
+bright central region of the source. It is possible to create a new trace, with the same 2D
+shape as the trace of the central region, but offset in the spatial direction. This might be useful
+for extracting spectra from the faint outer regions of the extended source, while using a trace computed from the
+brighter inner region. You can create a new trace based on the existing trace by clicking
+the "Trace" dropdown and selecting the existing trace. Then, offset it in the spatial direction by clicking or entering
+the spatial offset, and save it by creating a new trace or overwriting the existing trace entry.
```

Comment:
Enter it where? Is it obvious without more explanation?

Reply:
Yes I think it's clear, the offset should be entered into a box called "offset."

Reply:
my one thought/concern on this is that it is very tangential and specific to an "uncommon" workflow... but I don't have any ideas for where else to put this quite yet.